tensorly/backend/__init__.py

```diff
@@ -178,6 +186,8 @@ def inner(*args, **kwargs):
     return inner
 
 # Generic methods, exposed as part of the public API
+
+
```

Comment:
Why did you add these lines here after the comment?

Reply:
these lines created unintentionally.